backend/app/assets/stylesheets/spree/backend/shared/_forms.scss

```diff
@@ -306,3 +310,28 @@ fieldset {
 .form-buttons {
   text-align: center;
 }
+
+.form-section {
+  @include make-row(0px);
+  margin: 10px 0px;
+  padding: 10px 0px;
+  background: very-light($color-3, 4);
+  border: 1px solid very-light($color-3, 32);
+}
+
+.text-field-addon {
+  position: absolute;
+  right: 25px;
+  bottom: 21px;
+  pointer-events: none;
+}
+
+.max-width-form {
+  @include margin(null null auto null);
+  max-width: map-get($container-max-widths, "xl");
+}
```

Comment:
Maybe I miss something, but I think this is pretty much the role of the [`.container`](https://v4-alpha.getbootstrap.com/layout/overview/#containers) class. If I've guessed right, it will be great to get to use that. Otherwise, let me know what could be the problem with using it.